TrafficCapture/dockerSolution/README.md

```diff
@@ -14,6 +14,32 @@ While in the TrafficCapture directory, run the following command:
 
 `./gradlew :dockerSolution:composeUp`
 
+### Interacting with the Docker Solution
+
+A sample source cluster (running Elasticsearch 7.10) is set up with the Capture Proxy running in front of it. To route
+calls through the Capture Proxy, send requests to `localhost:9200` with the default credentials (admin, admin), e.g.:
+```sh
+curl --insecure https://localhost:9200/_cat/indices -u 'admin:admin'
+```
+
+You can send the same calls to the source cluster while bypassing the Capture Proxy (calls will not be relayed to the
+target cluster) via `localhost:19200`, and to the target cluster directly at `localhost:29200`.
+
+For sample data that exercises various endpoints with a range of datatypes, you can `ssh` into the Migration Console
```

Comment:
ssh threw me here.  There's no ssh involved.  You could say 'exec', but that might not be clear to somebody new to Docker.  Maybe s/ssh into/execute a shell within/ would be better.
ssh kept making me think that you were trying to hit some kind of a deployed cloud resource.